### PR TITLE
Convert 1st phase of continuous migration into support image

### PR DESCRIPTION
**Pull request by hjluo**
##
##### ${{\color{Purple}\Huge{\textsf{   Convert 1st phase of continuous migration into support image.\}}}}\$
- Related ticket: 
  * https://progress.opensuse.org/issues/153823
- Related MR:
  * https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/166/
- Needles: 
  * N/A
- Verification run: 
   * [**Support_image**](https://openqa.suse.de/tests/overview?arch=&flavor=&machine=&test=supp_migr_sles_15sp4_15sp5_dev&modules=&module_re=&group_glob=&not_group_glob=&comment=&version=15-SP5&groupid=318&distri=sle#)
   * Migration for 2nd step:
     [**Migration_VR**](https://openqa.suse.de/tests/overview?arch=&flavor=&machine=&test=migr_sles_continuous_15sp4_15sp5_ph1_dev1&modules=&module_re=&group_glob=&not_group_glob=&comment=&build=89.1&version=15-SP6&distri=sle#)
##

### CODE REVIEW

File: data/yam/autoyast/autoyast_scc_up.xml [hunk 1]
```diff
@@ -8,9 +8,8 @@
       <boot_mbr>true</boot_mbr>
       <boot_root>true</boot_root>
       <generic_mbr>false</generic_mbr>
-      <timeout config:type="integer">5</timeout>
+      <timeout config:type="integer">-1</timeout>
     </global>
-    <loader_type>{{LOADER_TYPE}}</loader_type>
```
Comment (lemon-suse) [inline data/yam/autoyast/autoyast_scc_up.xml:13]: Please reserve this line, it is for different loader type on different arches.

Comment (hjluo) [reply data/yam/autoyast/autoyast_scc_up.xml:13]: Now I'm set the `LOADER_TYPE=grub2` in the test case's setting.

Comment (hjluo) [reply data/yam/autoyast/autoyast_scc_up.xml:13]: @lemon-suse  now all VRs are passed, please take a look again, thanks!

**Review by lemon-suse — APPROVED** [review]

**Comment by hjluo** [comment]
> It's strange you haven't set Loader_type for aarch64 test but it passed, [openqa.suse.de/tests/14309561#](https://openqa.suse.de/tests/14309561#)
> 
> Please run a VR with correct loader_type.

Now after proxy SCC back to work and rerun all VR and passed.

**Comment by hjluo** [comment]
cause we are now reviewing GMC today, I'd suggest we hold it to be merged next week to avoid chaos in the group. Thanks

**Comment by lemon-suse** [comment]
> cause we are now reviewing GMC today, I'd suggest we hold it to be merged next week to avoid chaos in the group. Thanks

Ok, make sense.

**Pull request merged**